llm/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "Running job on `hostname`"
+echo "GPUs assigned: $CUDA_VISIBLE_DEVICES"
+
+export TRANSFORMERS_CACHE=$_CONDOR_SCRATCH_DIR/models
+export HF_DATASETS_CACHE=$_CONDOR_SCRATCH_DIR/datasets
+export HF_MODULES_CACHE=$_CONDOR_SCRATCH_DIR/modules
+export HF_METRICS_CACHE=$_CONDOR_SCRATCH_DIR/metrics
+
+# Update your run name here
+python3 train.py demo_run -w
```

Comment:
@ChristinaLK could we move this run name `demo_run` to be an argument in the submit file? That would be one less file that needs to be edited.